roles/custom/matrix-bridge-mautrix-wsproxy/templates/wsproxy-labels.j2

```diff
@@ -0,0 +1,45 @@
+{% if matrix_mautrix_wsproxy_container_labels_traefik_enabled %}
+traefik.enable=true
+
+{% if matrix_mautrix_wsproxy_container_labels_traefik_docker_network %}
+traefik.docker.network={{ matrix_mautrix_wsproxy_container_labels_traefik_docker_network }}
+{% endif %}
+
+{% set middlewares = [] %}
+
+{% if matrix_mautrix_wsproxy_container_labels_traefik_path_prefix != '/' %}
+traefik.http.middlewares.matrix-mautrix-wsproxy-slashless-redirect.redirectregex.regex=({{ matrix_mautrix_wsproxy_container_labels_traefik_path_prefix | quote }})$
+traefik.http.middlewares.matrix-mautrix-wsproxy-slashless-redirect.redirectregex.replacement=${1}/
+{% set middlewares = middlewares + ['matrix-mautrix-wsproxy-slashless-redirect'] %}
+{% endif %}
+
+{% if matrix_mautrix_wsproxy_container_labels_traefik_path_prefix != '/' %}
+traefik.http.middlewares.matrix-mautrix-wsproxy-strip-prefix.stripprefix.prefixes={{ matrix_mautrix_wsproxy_container_labels_traefik_path_prefix }}
+{% set middlewares = middlewares + ['matrix-mautrix-wsproxy-strip-prefix'] %}
+{% endif %}
+
+{% if matrix_mautrix_wsproxy_container_labels_traefik_additional_response_headers.keys() | length > 0 %}
+{% for name, value in matrix_mautrix_wsproxy_container_labels_traefik_additional_response_headers.items() %}
+traefik.http.middlewares.matrix-mautrix-wsproxy-add-headers.headers.customresponseheaders.{{ name }}={{ value }}
+{% endfor %}
+{% set middlewares = middlewares + ['matrix-mautrix-wsproxy-add-headers'] %}
+{% endif %}
+
+traefik.http.routers.matrix-mautrix-wsproxy.rule={{ matrix_mautrix_wsproxy_container_labels_traefik_rule }}
+{% if matrix_mautrix_wsproxy_container_labels_traefik_priority | int > 0 %}
+traefik.http.routers.matrix-mautrix-wsproxy.priority={{ matrix_mautrix_wsproxy_container_labels_traefik_priority }}
+{% endif %}
+traefik.http.routers.matrix-mautrix-wsproxy.service=matrix-mautrix-wsproxy
+{% if middlewares | length > 0 %}
+traefik.http.routers.matrix-mautrix-wsproxy.middlewares={{ middlewares | join(',') }}
+{% endif %}
+traefik.http.routers.matrix-mautrix-wsproxy.entrypoints={{ matrix_mautrix_wsproxy_container_labels_traefik_entrypoints }}
+traefik.http.routers.matrix-mautrix-wsproxy.tls={{ matrix_mautrix_wsproxy_container_labels_traefik_tls | to_json }}
+{% if matrix_mautrix_wsproxy_container_labels_traefik_tls %}
+traefik.http.routers.matrix-mautrix-wsproxy.tls.certResolver={{ matrix_mautrix_wsproxy_container_labels_traefik_tls_certResolver }}
+{% endif %}
+
+traefik.http.services.matrix-sliding-sync.loadbalancer.server.port=8008
```

Comment:
This is still wrong, haven't figured out how to get Traefik to forward to the right port like nginx does yet.

Reply:
Just submitted a PR to your fork with a fix!